lxd/patches.go

```diff
@@ -476,7 +486,15 @@ func upgradeFromStorageTypeBtrfs(name string, d *Daemon, defaultPoolName string,
 		if shared.PathExists(oldImageMntPoint) {
 			err := os.Rename(oldImageMntPoint, newImageMntPoint)
 			if err != nil {
-				return err
+				err = btrfsSnapshot(oldImageMntPoint, newImageMntPoint, false)
+				if err != nil {
+					return err
+				}
+				err = btrfsSubVolumesDelete(oldImageMntPoint)
+				if err != nil {
+					btrfsSubVolumesDelete(newImageMntPoint)
+					return err
+				}
```

Comment:
Hm, ah snapshots... So we could try to do a btrfssubvolumecreate and rsync over
but it might not be worth the trouble